crates/ide_db/src/helpers/insert_use/tests.rs

```diff
@@ -447,6 +447,16 @@ use std::io;",
     )
 }
 
+#[test]
+fn merge_groups_skip_attributed() {
+    check_full(
+        "std::io",
+        r#"#[cfg(feature = "gated")] use std::fmt::{Result, Display};"#,
```

Comment:
The test fixtures in this file don't use `r#` but I think they should (and they should start on the first column like in https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/style.md#minimal-tests).

You don't have to change this, though.

Reply:
I used `r#` because of the double quotes around the feature name. I then applied the "Format snippets more compactly" rule to make it a single line. This makes the `#"#` part look a bit weird and not immediately obvious to the reader, so it would indeed probably be clearer to split this over multiple lines.

Reply:
Sorry, I meant the line below.

Reply:
No problem. End result is the same, as newlines should match between the two anyway